docs/content/docs/5.v9/2.guide/18.breaking-changes-in-v9.md

```diff
@@ -46,4 +46,8 @@ nuxt.config.ts
 
 Reasons for change
   1. Context - i18n files are used both server-side and client-side, using a dedicated `i18n/` folder in the root directory outside `app/` and `server/` makes more sense.
-  2. Clean - less clutter/fragmentation of i18n files, and should make resolving and loading files easier for us.
\ No newline at end of file
+  2. Clean - less clutter/fragmentation of i18n files, and should make resolving and loading files easier for us.
+
+## Locale `iso` renamed to `language`
+
+The `iso` property on a locale object has been renamed to `language` as the original name refers to ISO standards which describe valid language tags.
```

Comment:
We should add more explanation why we made this change, include PR link.
This would make it easier for users to understand.
https://github.com/nuxt-modules/i18n/issues/2449